src/network/taraxa_capability.cpp

```diff
@@ -159,8 +161,9 @@ void TaraxaCapability::onConnect(weak_ptr<Session> session, u256 const &) {
     cnt_received_messages_[_nodeID] = 0;
     test_sums_[_nodeID] = 0;
 
-    insertPeer(_nodeID);
+    auto peer = insertPeer(_nodeID);
     sendStatus(_nodeID, true);
+    peer->sent_initial_status_ = true;
```

Comment:
we should get rid of this `peer->sent_initial_status_ ` and rather check if `sealAndSend` passes (check for true/false) and based on that mark things as known.

Reply:
Fixed.